src/pollManager.ts

```diff
@@ -49,4 +55,9 @@ export class PollManager {
     const fifoFilePath = this.config.get('uwsgiFifoFilePath');
     await $`echo r > ${fifoFilePath}`;
   }
+
+  public async delay(seconds: number): Promise<void> {
+    const msToSeconds = 1000;
```

Comment:
don't define it each time - change to static variable

Reply:
fixed